### ISSUE TITLE
upgrade from 8.1.5 to 8.1.6

### ISSUE THREAD

**Issue by centoasa** [PrestaShop/PrestaShop#36235]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

1. enable maintenance mode
2. enable debug mode
3. click on oneclick upgrade
4. I wait 50 seconds or more
5. here are the errors I get when upgrading from 8.1.5 to 8.1.6
6. `in [/home/udlah5r1/domains/100asa.it/public_html/classes/db/DbPDO.php ](https://www.100asa.it/Back100asa/index.php?controller=AdminSelfUpgrade&token=82f83b6322f9c89b5db8c3746c92922f)(line 151)
    protected function _query($sql)    {        try {            return $this->link->query($sql);        } catch (\PDOException $exception) {            throw new PrestaShopException($exception->getMessage(), (int) $exception->getCode(), $exception);        }    }    /**     * Returns the next row from the result s`
![screenshot-www 100asa it-2024 05 25-09_15_21](https://github.com/PrestaShop/PrestaShop/assets/5500935/69c68e71-e708-4956-8e00-e5244cca0547)

### Steps to reproduce

1. enable maintenance mode
2. enable debug mode
3. click on oneclick upgrade
4. I wait 50 seconds or more
5. here are the errors I get when upgrading from 8.1.5 to 8.1.6

### PrestaShop version(s) where the bug happened

8.1.5

### PHP version(s) where the bug happened

8.0 , 8.1

**Comment by AureRita** [comment]
Hi @centoasa 

Thank you for your report, Unfortunately, I can't reproduce your issue, can you provide us your log of your Autoupgrade ? 

@PrestaShop/squad-seamless-upgrade Did someone understand where is the issue with the provided screenshot ? 

Waiting for feedback

**Comment by Quetzacoalt91** [comment]
A few questions before starting the investigation:
* What error is triggered? The message has been truncated on the attached screenshot, it is displayed in the red header.
* When is this error happening, you're talking about the upgrade, but what were you doing at that moment? For instance did you refresh the page in the middle of the process or was it after a successful upgrade?
* As requested by @AureRita, can you provide the log of the upgrade process? It cas be found in your shop, `<admin_path>/autoupgrade/tmp/log.txt`. Before sharing it, make sure to replace all occurrences of your admin folder name by something else.

**Comment by centoasa** [comment]
triggered error is 500 error page!
when this error happens I am carrying out these steps (obviously after logging in as admin);
enable maintenance mode
enable debug mode
click on oneclick upgrade
I wait 50 seconds or more..
I've also disable all plugin (litespeed caceh for prestashop, cache and so on) clear cache.
Error (the first)
`in [/home/udlah5r1/domains/100asa.it/public_html/classes/db/DbPDO.php ](https://www.100asa.it/Back100asa/index.php?controller=AdminSelfUpgrade&token=82f83b6322f9c89b5db8c3746c92922f)(line 151)
    protected function _query($sql)    {        try {            return $this->link->query($sql);        } catch (\PDOException $exception) {            throw new PrestaShopException($exception->getMessage(), (int) $exception->getCode(), $exception);        }    }    /**     * Returns the next row from the result set.`

the log.txt file is always empty!

**Comment by Quetzacoalt91** [comment]
Thanks @centoasa,

By seeing how to reproduce your error, especially
>  I wait 50 seconds or more..

This is likely a duplicate of https://github.com/PrestaShop/PrestaShop/issues/36106, where many checks are being done before displaying the configuration page.
Some workarounds are provided in the other issue, and this is going to be fixed when the UI will be reworked.

Hope this helps.
